sqlstore/src/main/resources/db/migration/V8251__harmonize_TGL_kd_ts.sql

```diff
@@ -0,0 +1,43 @@
+--StartNoTest
+CALL addTargetedSequencing('Agilent SureSelectXT MethylSeq', 'TGL', 'Agilent SureSelectXT MethylSeq', 0);
+CALL addTargetedSequencing('IDT xGEN Exome Research Panel v1', 'TGL', 'IDT xGEN Exome Research Panel v1', 0);
+CALL addTargetedSequencing('TruSeq RNA Access', 'TGL', 'TruSeq RNA Access', 0);
+CALL addTargetedSequencing('Roche SeqCap Epi CpGiant', 'TGL', 'Roche SeqCap Epi CpGiant', 0);
+CALL addTargetedSequencing('TruSeq Methylation Epic Exome', 'TGL', 'TruSeq Methylation Epic Exome', 0);
+
+CALL addKitDescriptor('Agilent SureSelect XT Human All Exon V5 + UTRs', 1, 'Agilent', 1, 'Library', 'Illumina', 'TGL');
+
+UPDATE KitDescriptor SET name = 'Agilent SureSelect XT Human All Exon V6 Cosmic' WHERE name = 'Agilent SureSelect Human All Exon V6 Cosmic';
```

Comment:
Kit 'Agilent SureSelect Human All Exon V6 Cosmic' exists in GSLE, so if it is renamed, there will be issues with Pinery-MISO migration, and a mismatch between GSLE data pre migration and MISO data post migration